packages/@jsii/go-runtime/jsii-runtime-go/runtime.go

```diff
@@ -359,3 +359,8 @@ func convertArguments(args []interface{}) []interface{} {
 func Close() {
 	kernel.CloseClient()
 }
+
+// Helpers to store primitives and return pointers to them
+func Bool(v bool) *bool         { return &v }
+func Number(v float64) *float64 { return &v }
+func String(v string) *string   { return &v }
```

Comment:
We need to either move those to another package, or move the runtime API (except for `Close`) to a non-internal sub-package, and use it from there...

Right now - this likely will make people itch to use toys we don't want them to play with.

Reply:
Wrote a ticket to extract user land functions into a separate package. https://github.com/aws/jsii/issues/2721